docs/index.bs

```diff
@@ -3873,6 +3926,27 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Return true.
       1. Return false.
   </section>
+
+  <section algorithm>
+    <h3 id="lookup-race-response-algorithm"><dfn export>Lookup Race Response</dfn></h3>
+
+      : Input
+      :: |request|, a [=request=]
+      : Output
+      :: a [=race response=].
```

Comment:
As this is called from the fetch spec, if possible, perhaps it's convenient to set just `"<code>pending</code>", [=/response=], or null` as an output.

Reply:
Yes, I think unwrapping the struct would be better, so that the Fetch spec doesn't need to know about it. (And then you can avoid `export`ing it.)

Also, is Fetch really going to treat all three outcomes differently? Or maybe if it's still pending at the time of the call, it will treat pending the same as null? If so the you could handle that in this algorithm too.

Reply:
Thanks! Updated to return [=/response=] or null. Fetch is going to treat if there is an map entry, which returns [=/response=], or no entry.